internal/upstreamldap/upstreamldap.go

```diff
@@ -615,6 +624,7 @@ func (p *Provider) searchAndBindUser(conn Conn, username string, bindFunc func(c
 			Name:   mappedUsername,
 			UID:    mappedUID,
 			Groups: mappedGroupNames,
+			Extra:  mappedRefreshAttributes,
```

Comment:
Do not use the `user.Info.Extra` field, it has a different meaning.  Create a new field as we did for `DN`.